### PR TITLE
Pull upstream changes

### PR DESCRIPTION
**Pull request by mcdonnnj**
This PR just pulls in the latest changes from the `skeleton-ansible-role` repository.

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@ platforms:
     image: debian:stretch-slim
   - name: debian10
     image: debian:buster-slim
+  - name: fedora29
+    image: fedora:29
+  - name: fedora30
+    image: fedora:30
+  - name: amazon2018.03
+    image: amazonlinux:2018.03
```
Comment (dav3r) [inline molecule/default/molecule.yml]: Did you test this role on all of these operating systems?  If not, feel free to remove them.

**Review by dav3r — APPROVED** [review]
No major issues, but please please see my one comment.

File: molecule/default/molecule.yml (hunk 1 above)
Comment (mcdonnnj) [reply molecule/default/molecule.yml]: Pared it down to just Debian. Thanks for the feedback. 👍

Comment (mcdonnnj) [reply molecule/default/molecule.yml]: Pared it down to just Debian. Thanks for the feedback. 👍

**Review by jsf9k — APPROVED** [review]
Powerful work

**Pull request merged**